TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Timeline/Events/RaisedByUser.cs

```diff
@@ -0,0 +1,9 @@
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.Timeline.Events;
+
+public record RaisedByUser
+{
+    public required User? TrsUser { get; set; }
```

Comment:
In `RaisedByUserInfo` we've not used a `Trs` prefix
```suggestion
    public required User? User { get; set; }
```

Reply:
Done